src/assets/scripts/client/index.js

```diff
@@ -3,30 +3,19 @@ require('raf').polyfill();
 import 'babel-polyfill';
 import $ from 'jquery';
 import App from './App';
-import AirportController from './airport/AirportController';
 import _has from 'lodash/has';
 import _lowerCase from 'lodash/lowerCase';
 import { DEFAULT_AIRPORT_ICAO } from './constants/airportConstants';
 import { STORAGE_KEY } from './constants/storageKeys';
 
-const getInitialAirport = () => {
-    let airportName = DEFAULT_AIRPORT_ICAO;
-
-    if (_has(localStorage, STORAGE_KEY.ATC_LAST_AIRPORT) && AirportController.hasAirport(STORAGE_KEY.ATC_LAST_AIRPORT)) {
-        airportName = _lowerCase(localStorage[STORAGE_KEY.ATC_LAST_AIRPORT]);
-    }
-
-    return airportName;
-};
-
 /**
  * Entry point for the application.
  *
  * Provides a way to grab the `body` element of the document and pass it to the app.
  */
 export default (() => {
     const airportLoadList = window.AIRPORT_LOAD_LIST;
-    const initialAirportToLoad = getInitialAirport();
```

Comment:
we still need this function in order to get any `localStorage` value.  the new code you've added within the `$.when()` should hopefully handle cases where that value doesn't exist.